partners/scientific-communities.md

```diff
@@ -1,38 +1,47 @@
-# Partnerships with Scientific Python Communities
+# Partnerships with scientific Python Communities
 
-```{toctree}
+
+
+:::{toctree}
 :hidden:
 :caption: Our Partners
 
 About Our Partnerships <self>
 JOSS <joss>
 Pangeo <pangeo>
-```
+:::
+
 
-Numerous scientific Python communities are developing their own peer review
-processes for affiliated packages. Communities with their own package
-review processes can leverage our existing peer review process, which is
-built on community-driven package standards. In partnership with these
-communities, we enhance our standard review process with a community-specific
+:::{figure-md} pyopensci-3-pillars
+
+<img src="../images/pyopensci-pillars-flower.png" alt="Flower" width="500">
+
+pyOpenSci has three core programs. Community partnerships is a program that allows domain specific scientific Python communities to leverage our peer review process rather than creating their own process. Through this partnership maintainers can become affiliated with both a domain specific community, published in the Journal of Open Source Software and also become a part if the pyOpenSci community.
```

Comment:
```suggestion
pyOpenSci has three core programs: program one, program two, program three. Community partnerships allow domain-specific scientific Python communities to leverage the pyOpenSci peer review process rather than creating their own process. Through this partnership, maintainers can become affiliated with a domain-specific community, published in the Journal of Open Source Software (JOSS) and join the pyOpenSci community.
```

Reply:
@kierisi your in place text change makes the text a bit confusing with program one, program two etc. are you suggesting that  i list the name of each program there? or we could add numbers before each section? i just wouldn't want generic text.

Reply:
if you wanted me to insert the program names a nice approach could be to only edit the text inline where you are confident in actually changing the text (if i merge this the text will now say program one, program two, etc) . then add a note to the suggested change that says - i'd spell out the program names or something like that. because as is i can't merge this (unless i am mis understanding the intent that is!)

Reply:
ok i have. just rewritten this all alt tag and caption so it's more clear. resolving!